src/pages/Routes.tsx

```diff
@@ -58,98 +58,96 @@ const Routes = () => {
+            <Text bold fontSize='5xl'>Active Routes</Text>
+            <Text fontSize='4xl'>Error loading routes</Text>
+          </Box>
+          <Box w='1/2' alignItems='center'>
+            <Text bold fontSize='5xl'>Archived Routes</Text>
+            <Text fontSize='4xl'>Error loading routes</Text>
+          </Box>
+        </HStack>
+      </Center>
     );
   }
   return (
-    <VStack height='100%'>
-      <Button onPress={() => setCreateRoutePopup(true)} mt='20'>
-        <Text variant='button'>Create route</Text>
-      </Button>
-      <Flex flexDirection='row' justifyContent='space-evenly' width='100%' top='50px'>
-        <Flex flexDirection='row' justifyContent='center' width='30%'>
-          <Flex flexDirection='column' alignItems='center' width='100%'>
-            <Text bold fontSize='lg'>Active Routes</Text>
+    <Box>
+      <Center>
+        {/* TODO: add 'are you sure?' popup to buttons */}
+        <HStack p='3' space='lg'>
+          <Button onPress={() => setCreateRoutePopup(true)}>
+            <Text variant='button'>Create route</Text>
+          </Button>
+          <Button onPress={() => archiveAllOfType(RouteType.Boulder)}>
+            <Text variant='button'>Archive All Boulders</Text>
+          </Button>
+          <Button onPress={() => archiveAllOfType(RouteType.Traverse)}>
+            <Text variant='button'>Archive All Traverses</Text>
+          </Button>
+          <Button onPress={() => archiveAllOfType(RouteType.Toprope)}>
+            <Text variant='button'>Archive All Top Ropes</Text>
+          </Button>
+        </HStack>
+      </Center>
+
+      <Center>
+        <HStack justifyContent='space-around' w='4/6' space='16'>
+          <Box w='1/2' alignItems='center'>
+            <Text bold fontSize={{
+              base: 'lg',
+              lg: '2xl',
+              xl: '4xl'
+            }}>Active Routes</Text>
             {
               data.activeRoutes.map((currRoute: Route) => (
                 <VStack key={currRoute.docRef?.id} width='100%'>
-                  <Divider orientation='horizontal' height='2px' />
+                  <Divider thickness='3' />
                   <RouteRow route={currRoute} />
                 </VStack>
               ))
             }
-          </Flex>
-        </Flex>
-        <Divider orientation='vertical' height={'65vh'} position='fixed' />
-        <Flex flexDirection='row' justifyContent='center' width='30%'>
-          <Flex flexDirection='column' alignItems='center' width='100%'>
-            <Text bold fontSize='lg'>Archived Routes</Text>
+          </Box>
+          <Box w='1/2' alignItems='center'>
+            <Text bold  fontSize={{
+              base: 'lg',
+              lg: '2xl',
+              xl: '4xl'
+            }}>Archived Routes</Text>
             {isLoading ? <Text>Loading...</Text> :
               archivedRoutes.map((currRoute: Route) => (
                 <VStack key={currRoute.docRef?.id} width='100%'>
```

Comment:
I think this VStack should be a Scroll View come to think of it? or will it automatically allow us to scroll the whole page?

Reply:
Sorry I originally didn't see this. It'll just let us scroll through the whole page. Scrollview's are specifically for react native